mithril-test-lab/benchmark/aggregator-prover/benchmark-aggregator-prover.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+set -e
+
+# Debug mode
+if [ -v DEBUG ]; then
+    set -x
+fi
+
+# Check if all env vars are set 
+if [ -z "${AGGREGATOR_ENDPOINT}" ]; then
+    echo "Missing environment variable: AGGREGATOR_ENDPOINT" >/dev/stderr
+    exit 1
+fi
+
+if [ -z "${TRANSACTIONS_FILE}" ]; then
+    echo "Missing environment variable: TRANSACTIONS_FILE" >/dev/stderr
+    exit 1
+fi
+
+if [ -z "${TRANSACTIONS_PER_REQUEST_MIN}" ]; then
+    TRANSACTIONS_PER_REQUEST_MIN=0
+fi
+
+if [ -z "${TRANSACTIONS_PER_REQUEST_MAX}" ]; then
+    TRANSACTIONS_PER_REQUEST_MAX=100
+fi
+TRANSACTIONS=$(cat $TRANSACTIONS_FILE | tr "\n" " ")
+TRANSACTIONS_AVAILABLE=$(echo $TRANSACTIONS | wc -w)
```

Comment:
Suggestion to Use to remove unnecessary spaces before and after the number:
```suggestion
TRANSACTIONS_AVAILABLE=$(echo $TRANSACTIONS | wc -w | xargs)
```

Before
`>> Transactions available: [     100]`

After:
`>> Transactions available: [100]`

Reply:
Done